test/array.specs.ts

```diff
@@ -0,0 +1,81 @@
+import "mocha";
+import fc from "fast-check";
+import { chunk } from "../src/array"; 
+
+describe("chunk: ", () => {
+    it("Should split arrays into chunks of the specified size", () =>
+      fc.assert(
+        fc.property(
+          fc.array(fc.integer(), { minLength: 1 }), //tableau non vide
+          fc.nat(10), // la taille de chunk entre 0 et 10
+          (array, size) => {
+            const result = chunk(array, Math.max(size, 1)); // garantir que la taille du chunk est au moins 1
+            return result.length > 0 && result.every((chunk) => chunk.length <= Math.max(size, 1));
```

Comment:
All chunks should have the same size except maybe the last one and that size should be `size`.
You dont need to test its length, but you do need to check that it's precisely size for all except the last one and that the last one is <= to size